services/QuillLMS/app/services/vitally_integration/serialize_vitally_sales_organization.rb

```diff
@@ -0,0 +1,194 @@
+# frozen_string_literal: true
+
+class SerializeVitallySalesOrganization
```

Comment:
Note: This class's purpose is serialization, but it does not inherit from ApplicationSerializer, nor does it live in `serializers/`.  

I'm sort of OK leaving it as-is, but it's worth highlighting.

Reply:
I'm following the naming pattern we use for all the Vitally "serializers" (there's two others by that name in the folder, for schools and teachers). 

Good to note this -- I'll keep it for now, but maybe we can do a renaming next time there's additions to Vitally "serializers" here and rename all of these classes.